sublime-package.json

```diff
@@ -0,0 +1,738 @@
+{
+  "contributions": {
+    "settings": [
+      {
+        "file_patterns": [
+          "/LSP-intelephense.sublime-settings"
+        ],
+        "schema": {
+          "$id": "sublime://settings/LSP-intelephense",
+          "definitions": {
+            "PluginConfig": {
+              "properties": {
+                "initializationOptions": {
+                  "additionalProperties": false,
+                  "properties": {
+                    "clearCache": {
+                      "type": "boolean",
+                      "default": false,
+                      "description": "Clear server state on startup."
+                    },
+                    "globalStoragePath": {
+                      "type": "string",
+                      "default": "${home}/.intelephense",
+                      "description": "Absolute path to global dir. Defaults to `os.homedir()`."
```

Comment:
When using markdown (backticks), it will look better if you use `markdownDescription` property.